security/middleware.py

```diff
@@ -198,14 +243,14 @@ def process_view(self, request, view, *args, **kwargs):
             url_name = resolve(request.path_info).url_name
 
             # Check for an exempt URL before trying to resolve URL_NAME,
-            # because the reason the URL is exempt may be because a special
-            # URL config is in use (i.e. during a test) that doesn't have URL_NAME.
-            if (not request.user.is_authenticated() or
-                        view == django.views.static.serve or  # Mostly for testing, since
-                # Django shouldn't be serving
-                # media in production.
-                    any(m.match(path) for m in self.exempt_urls) or
-                        url_name in self.settings.get("EXEMPT_URL_NAMES", ())):
+            # because the reason the URL is exempt may be because a special URL
+            # config is in use (i.e. during a test) that doesn't have URL_NAME.
+            if (
```

Comment:
I just noticed your other pull request.

Good point about diffs. Don't feel like you need to make changes just because. I just noticed stuff that jumped out at me.